xclim/sdba/utils.py

```diff
@@ -13,7 +13,9 @@
 import xarray as xr
 from boltons.funcutils import wraps
 from dask import array as dsk
+from ot import emd
```

Comment:
We should discuss at some point:
* if we want to include this method in SDBA by default, you should add `ot` as a requirement
* If we leave dOTC optional, you could take a look at SBCK's import and put conditional import for `ot`, and your functions/classes